src/containers/Pages/AboutPage/AboutPage.js

```diff
@@ -1,17 +1,78 @@
 import React from "react";
 import styles from "./AboutPage.module.scss";
+import MESSAGE from "../../../constants/Messages";
+import Profiles from "../../../constants/Profiles.js";
+import emailIcon from "../../../assets/icons/email-logo.svg";
+import githubIcon from "../../../assets/icons/github-logo.svg";
+import linkedinIcon from "../../../assets/icons/linkedin-logo.svg";
+import NavBarWSeachbar from "../../../components/NavBarWSearchbar/NavBarWSeachbar";
+import Aux from "../../../hoc/auxilary";
 
 class AboutPage extends React.Component {
   render() {
-    return (
-      <div className={styles.content}>
-        <h1>Name</h1>
-        <h1>Pic</h1>
-        <h1>History</h1>
-        <h1>etc</h1>
-        <h1>....</h1>
+    const ABOUT_US = (
+      <div className={styles.aboutUs}>
+        <h1>{MESSAGE.about_page_title}</h1>
+        <h2>{MESSAGE.about_page_subtitle}</h2>
+        <p>{MESSAGE.about_page_description1}</p>
+        <p>{MESSAGE.about_page_description2}</p>
       </div>
     );
+
+    const TEAM = Profiles.map((teamMember, index) => {
+      return (
+        <div
+          className={styles.magnifyGlassContainer}
+          key={teamMember.name + index}
+        >
+          <div className={styles.magnifyGlass}>
+            <img
+              src="https://www.w3schools.com/howto/img_avatar.png"
+              alt="test"
+              className={styles.imgAvatar}
+            ></img>
+            <div className={styles.logoAvatarContainer}>
+              <a href="https://google.ca">
+                <img
+                  src={emailIcon}
+                  className={styles.avatarIcon}
+                  alt="Email Icon"
+                ></img>
+              </a>
+              <a href="https://google.ca">
+                <img
+                  src={githubIcon}
+                  className={styles.avatarIcon}
+                  alt="Github Icon"
+                ></img>
+              </a>
+              <a href="https://google.ca">
+                <img
+                  src={linkedinIcon}
+                  className={styles.avatarIcon}
+                  alt="LinkedIn Icon"
+                ></img>
+              </a>
+            </div>
+            <div className={styles.magnifyGlassHandle}></div>
+          </div>
+          <div className={styles.teamMemberInfo}>
+            <h4>{teamMember.name}</h4>
+            <p>{teamMember.description}</p>
+          </div>
+        </div>
+      );
```

Comment:
i think all of this needs go into a component by itself

Reply:
ack